openapi/data_repository_service.swagger.yaml

```diff
@@ -396,3 +406,33 @@ definitions:
       license:
         type: object
         description: License information for the exposed API
+  BundleObjects:
+    type: object
+    properties:
+      name:
+        type: string
+        description: >-
+          This represents the Bundle author declared name that must be
+          used when materialising the associated data object and will 
+          override any name directly associated with the object itself.
+          This string MUST NOT contain any slashes.
+      id:
+        type: string
+        description: >-
+          This represents the DRS identifier that may point to a Data Object
```

Comment:
Is DRS identifier well-defined somewhere? (Easily could be. I'm still finding my way around here...) I assume this is intended to be a complete DRS URI - not just the id component. Is that correct?

Reply:
Yes, it should be the complete DRS URI. I will clarify the description.

I think the DRS ID URI must be defined somewhere in the documentation.